recipe/meta.yaml

```diff
@@ -49,6 +48,7 @@ about:
     for requests made to Anaconda Cloud services.
   home: https://anaconda.cloud
   doc_url: https://pypi.org/project/anaconda-cloud-auth/
+  dev_url: https://pypi.org/project/anaconda-cloud-auth/
```

Comment:
```suggestion
  dev_url: https://github.com/anaconda/anaconda-cloud-tools/tree/main/libs/anaconda-cloud-auth
```

Reply:
this is an internal URL though so check whether we want to expose it in a public recipe

Reply:
We shouldn't expose it.